src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -58,6 +58,18 @@ trans, quat, mc), which some installations impose to be run on raw data
 because of active shielding software corrections before the MEG data can
 actually be exploited.
 
+### Recording EEG simultaneously with MEG
+
+Note that if EEG is recorded with a separate amplifier,
+it SHOULD be stored separately under a new `/eeg` data type
+(see [the EEG specification](03-electroencephalography.md)).
+
+If however EEG is recorded simultaneously **with the same MEG system**,
+it MAY be stored under the `/meg` data type.
+In that case, it SHOULD have the same sampling frequency as MEG (see `SamplingFrequency` field below).
+Furthermore, the EEG sensor coordinates SHOULD be specified using MEG specific coordinate
+systems (see coordinates section below and [Appendix VIII](../99-appendices/08-coordinate-systems.md)).
```

Comment:
Which section would that be? Can we link to it?

Reply:
yes, should be like:

```suggestion
systems (see [coordinates section](#coordinate-system-json-coordsystem-json) below and [Appendix VIII](../99-appendices/08-coordinate-systems.md)).
```